build-vm-kvm

```diff
@@ -275,9 +276,6 @@ vm_startup_kvm() {
     qemu_args=(-drive file="$VM_ROOT",format=raw,if=none,id=disk,cache=unsafe -device "$kvm_device",drive=disk,serial=0)
     if [ -n "$VM_USER" ] ; then
 	getent passwd "$VM_USER" > /dev/null || cleanup_and_exit 3 "cannot find KVM user '$VM_USER'"
-    else
-	# use qemu user by default if available
-	getent passwd qemu >/dev/null && VM_USER=qemu
```

Comment:
we could instead wrap this in a `if [ $UID = 0 ] ; then`

Reply:
yep, we will add that